packages/coding-standards/easy-coding-standard.yml

```diff
@@ -128,6 +128,8 @@ services:
         maxCount: 30
 
 parameters:
+    exclude_files:
+        - '*/tests/Unit/**/wrong/*'
```

Comment:
I have a question - is this file used by other packages? If so, then this line would ignore files in other packages, and that is not prefered in my opinion. Am I right? If so, is it possible to solve excluding wrong tests in a different way? Because I don't know

Reply:
this file is used only by monorepo and coding-standards package in other packages.
this kind of solution is used in other packages, path is almost specific so it should not be problem in most cases as general and no problem in out monorepo.